### PR TITLE
add function to buy "Streak on Ice"

### PR DESCRIPTION
**Pull request by andreasscherbaum**
Update documentation for shop

### CODE REVIEW

File: duolingo.py [hunk 1]
```diff
@@ -80,9 +80,41 @@ def buy_item(self, item_name, abbr):
...
+            # any other error:
+            raise Exception('Not possible to buy item.')
+
+    def buy_streak_freeze(self):
+        """
+        figure out the users current learning language
+        use this one as parameter for the shop
+        """
+        lang = self.get_abbreviation_of(self.get_user_info()['learning_language_string'])
+        if (lang is None):
```
Comment (KartikTalwar) [inline duolingo.py]: can you remove the brackets in this line and L122

**Comment by KartikTalwar** [comment]
@andreasscherbaum happy to merge this after the conflict is resolved

**Comment by andreasscherbaum** [comment]
@KartikTalwar That last commit should do it, I believe.

I've seen that you added get_leaderboard() in README.md, but it's not in the function list at the beginning of the file.

**Comment by KartikTalwar** [comment]
@andreasscherbaum thanks for catching that. Updating readme

**Pull request merged**